README.md

```diff
@@ -83,8 +95,12 @@ this is a pretty good [resource](https://docs.docker.com/engine/examples/postgre
 
 # Testing
 
-Run unit tests from the root of the repo
-`python3 -m unittest`
+Set up a test database by running the database migrations and setting the `TEST_DATABASE_URL` in the
+`.env` file.
+
+Run unit tests and Snapshot tests for GraphQL.
+
+`nosetests`
```

Comment:
Is this the command for testing gql and did we remove the unit test commands?

Reply:
@13ABEL `nosetests` also executes unit tests. I'm setting up Postman right now which will require a different command for running test and will update this once I'm done.